Tests/LibWasm/test-wasm.cpp

```diff
@@ -4,23 +4,30 @@
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
-#include <LibCore/File.h>
+#include <LibCore/Stream.h>
 #include <LibTest/JavaScriptTestRunner.h>
 #include <LibWasm/AbstractMachine/BytecodeInterpreter.h>
 #include <LibWasm/Types.h>
-#include <string.h>
```

Comment:
`string.h` is still used directly here for `strerror`

Reply:
:+1: Thanks